clickable.py

```diff
@@ -1214,17 +1211,128 @@ def _build(self):
         self.run_container_command(gocommand)
 
 
-class CordovaClickable(Clickable):
+class CordovaClickable(CMakeClickable):
+    # Lots of this code was based off of this:
+    # https://github.com/apache/cordova-ubuntu/blob/28cd3c1b53c1558baed4c66cb2deba597f05b3c6/bin/templates/project/cordova/lib/build.js#L59-L131
+    def __init__(self, *args, **kwargs):
+        super(CMakeClickable, self).__init__(*args, **kwargs)
+
+        self.platform_dir = os.path.join(self.cwd, 'platforms/ubuntu/')
+
+        self._dirs = {
+                'build'  : '{}/{}/{}/build/' .format(self.platform_dir, self.config.sdk, self.build_arch),
+                'prefix' : '{}/{}/{}/prefix/'.format(self.platform_dir, self.config.sdk, self.build_arch),
+                'make'   : '{}/build'.format(self.platform_dir)
+        }
+
+        self.temp = self._dirs['build']
+
+        self.config.specificDependencies = True
+        self.config.dependencies = [ # Got this list from check_reqs or somewhere
```

Comment:
Looking at the [dockerfile](https://github.com/bhdouglass/clickable/blob/master/docker/ubuntu-sdk/15.04-armhf/Dockerfile), the only identical shared package is `cmake`. Two others that are somewhat similar are `pkg-config` with `pkg-config-arm-linux-gnueabihf` and `qtdeclarative5-dev:armhf` with ` qtdeclarative5-private-dev:armhf`

I think it's reasonable to remove `cmake`(as the cordova class subclasses the cmake class, it's fair to assume cmake is installed), but as for the others is there much of a downside?

Reply:
Most of those should be installed as dependencies of the `ubuntu-sdk-libs` meta package. If they aren't included already then we should add them to the docker image. Adding them to the docker image will speed up development for user (because if there are any dependencies listed clickable will generate a new docker image for those dependencies).

Reply:
I've trimmed down the list to only packages that aren't installed in the docker container by default. I'll remove it once the container is updated but I'm not comfortable with an open pull request that flat out doesn't work.

Reply:
I'll agree with that :+1: